packages/hash/integration/src/mockData/index.ts

```diff
@@ -492,6 +482,34 @@ void (async () => {
     ]),
   );
 
+  const table1 = results.get("t1");
+  const table1AccountId = table1?.accountId;
+  const table1EntityId = table1?.entityId;
+
+  if (table1AccountId && table1EntityId) {
+    await db.transaction(async (client) => {
+      const source = await Entity.getEntityLatestVersion(client, {
+        accountId: table1AccountId,
+        entityId: table1EntityId,
+      });
+
+      await source?.createAggregation(client, {
+        stringifiedPath: "$.data",
+        operation: {
+          entityTypeId: newTypeIds.Person,
+          itemsPerPage: 5,
+          multiSort: [
+            {
+              field: "createdAt",
+            },
+          ],
+          pageNumber: 1,
+        },
+        createdBy: user,
+      });
```

Comment:
**Potentially blocking:** is it expected behaviour for the table block to be empty when the database is seeded? From the way I read this I would expect the table to display the list of people, but instead I get:
<img width="660" alt="image" src="https://user-images.githubusercontent.com/42802102/155160858-22585654-0e0c-4f32-b060-69ff00db0656.png">

Reply:
Do you have any errors while running `yarn seed-data`?

I'm seeing the table block correctly, so let me know if you want to pair on this.

Reply:
I don't encounter errors whilst running `yarn seed-data`, if you're getting a loaded table block after `yarn seed-data` and I'm not perhaps we should pair on this yes.

Reply:
I do see a block with data after running `yarn seed-data`, yeah.

Reply:
Are you both looking at the same page? The table block with `Person` only appears on the test page titled "My awesome page". Ben appears to be looking at a different page.

Tables load with no entity type selected by default.

Reply:
I'm not getting tables on any pages (under the account "alice"). To be clear, my "My Awesome Page" looks like this:
<img width="1488" alt="image" src="https://user-images.githubusercontent.com/42802102/155299663-72e87ab2-4580-4da8-9d0c-850d8fad28b7.png">
And the table testing page looks like:
<img width="1850" alt="image" src="https://user-images.githubusercontent.com/42802102/155299708-519b71c5-19cc-47fa-bf24-732dcbe2ccd1.png">

Reply:
We resolved this in a huddle, it was an issue with the wrong block being loaded in my local environment.